sys-firmware/intel-wifi-firmwares/intel_wifi_firmwares-2022_01_10p0.recipe

```diff
@@ -5,18 +5,18 @@ contained within the base system."
 HOMEPAGE="https://www.intel.com/content/www/us/en/support/articles/000005511/wireless.html"
 COPYRIGHT="2006-2022 Intel Corporation"
 LICENSE="Intel WiFi Firmware"
-REVISION="2"
+REVISION="3"
```

Comment:
reset this to 1 and drop the `p0` from the filename

Reply:
The version goes from 2022_01_11 to 2022_01_10p0, isn't this a downgrade?

Reply:
Ah, yeah, whoops, I missed that.  
So indeed the old filename should stand after all, and the revision bump stay.